docs/FAQ.md

```diff
@@ -16,6 +16,9 @@ During rpm-ostree operations, it's normal. Outside of that, make sure you follow
 
 [No](https://madaidans-insecurities.github.io/linux.html#firejail), use ``bubblejail`` if there's no flatpak available for an app.
 
+#### Help! My browser won't start!
```

Comment:
Please phrase it in the form of a question like the other entries, like:


#### Why won't chromium start?